docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -75,26 +71,26 @@ This sample demonstrates four Model in Image use cases:
      - `spec.model.configMap` referencing the ConfigMap
      - References to data source secrets
 
-- [Update2](#update2-use-case): Demonstrates deploying a second domain (similar to the Update1 use case domain).
+- [Update 2]({{< relref "/samples/simple/domains/model-in-image/update2.md" >}}): Demonstrates deploying a second domain (similar to the Update 1  domain).
 
   - Image `model-in-image:WLS-v1`:
-    - Same image as the Initial and Update1 use cases
+    - Same image as the Initial and Update 1 use cases
   - Kubernetes Secrets and ConfigMap:
-    - Similar to the Update1 use case, except names and labels are decorated with a new domain UID
-  - A domain resource, similar to Update1 use case, except:
+    - Similar to the Update 1 use case, except names and labels are decorated with a new domain UID
+  - A domain resource, similar to Update 1 use case, except:
     - Its `metadata.name` and `weblogic.domainUid` label become `sample-domain2` instead of `sample-domain1`
     - Its secret/ConfigMap references are decorated with `sample-domain2` instead of `sample-domain1`
     - Has a changed `env` variable that sets a new domain name
 
-- [Update3](#update3-use-case): Demonstrates deploying an updated image with an updated application to the Update1 use case domain.
+- [Update 3]({{< relref "/samples/simple/domains/model-in-image/update3.md" >}}): Demonstrates deploying an updated image with an updated application to the Update 1  domain.
 
   - Image `model-in-image:WLS-v2`, similar to `model-in-image:WLS-v1` image with:
     - An updated web application `v2` at the `myapp-v2` directory path instead of `myapp-v1`
     - An updated model that points to the new web application path
   - Kubernetes Secrets and ConfigMap:
-    - Same as the Update1 use case
+    - Same as the Update 1 use case
   - A domain resource:
-    - Same as the Update1 use case, except `spec.image` is `model-in-image:WLS-v2`
+    - Same as the Update 1 use case, except `spec.image` is `model-in-image:WLS-v2`
 
 #### Sample directory structure
```

Comment:
Might as well split out pre-req-wls pre-req-jrf and initial into different files too?

Reply:
We need to strike a balance between navigating to separate files versus not. I kept all the prereqs together with the initial use case for this reason: if a user wants to run through just the first scenario, it's all contained in one file.

Reply:
Hmm, I still lean towards separating them. IMO, bundling overview + pre-reqs + initial together makes for an overly long file in of itself. Also, I find it a little confusing that some of the use cases are included in the introduction document, but not all of them.

Reply:
Perhaps, I should break out all the use cases into separate files and keep the intro and prereqs together.

Reply:
I personally would prefer an "overview" top level doc that points to "intro" "pre-reqs" "initial" "update1" "update2" "update3" and "cleanup" - similar to the organization of the MII user docs   In for a penny, in for a pound.   Maybe wouldn't look too overwhelming if if there's a way to suppress all but "overview" in hugo's left side panel menu.

Reply:
I think I will try something along these lines because the doc is so long. I don't think that there is a way to temporarily suppress entries in Hugo's left-side panel menu. However, it might be possible to make it expandable/collapsible (but I don't currently know how to do that).